docs/reference/index-modules/similarity.asciidoc

```diff
@@ -494,36 +504,48 @@ it is <<indices-create-index,created>>:
 
 [source,js]
 --------------------------------------------------
-PUT /my_index
+PUT index
```

Comment:
I've mostly been starting with the `/` before the first element so it feels more like a url. At first I liked it better without but @clintongormley liked it better with so I went with his way and I've grown used to it.

Reply:
Sadly we're not coherent in the docs. I changed it here so that it is coherent on the same page.